src/mergeSort.js

```diff
@@ -0,0 +1,27 @@
+export default function mergeSort(array) {
+
+  if(!Array.isArray(array)) return "We can only sort arrays"
+  if(array.length < 2) return array
+  const middle = Math.floor(array.length / 2)
```

Comment:
Consider the use of whitespace in between the error checking and setup to make this a little easier to read, and to provide a clear visual signal that there is a change in logic.